### DISCUSSION TITLE
Deeplinks don't support the extension?

### DISCUSSION THREAD

**Discussion by Mattijah** [phantom/docs#248] [General]
So I'm not sure if this is a bug or a feature not supported. I am clicking on a deeplink that can be found in the bottom here https://docs.phantom.app/phantom-deeplinks/other-methods/browse (or even my own ones) and it opens page saying "_Opening link..._", couple more seconds in and I get redirected to the download page. Is the extension wallet not supported?

**Comment by adamdelphantom** [comment]
Hello @Mattijah,

Your understanding is correct. Deeplinks are only designed for and supported on mobile platforms.

**Comment by Mattijah** [comment]
Are there any plans for adding this feature? Standalone apps are otherwise unable to establish a connection on desktop platforms. Or how else could I resolve this?

**Reply by adamdelphantom** [reply]
No plans for this, currently. 

We recently announced plans for an embedded wallet along with our acquisition of Bitski. More info [here](https://x.com/phantom/status/1792903231774834689).

We don't have an eta to share for embedded wallets but I think this will fit your use case best once it is available.
